htdocs/api/v0.0.2/candidates/Visit.php

```diff
@@ -15,10 +15,12 @@
     get_include_path()
     . ":" . __DIR__ . "/../"
     . ':' . __DIR__ . "/../../../../php/libraries/"
+    . ':' . __DIR__ . "/../../../../modules/create_timepoint/php/"
```

Comment:
Requires into a different module adds a bit of a code and has been fickle when we've done it before.

Wouldn't it make more sense to move the validation logic to the TimePoint class and call it from both places?